### DISCUSSION TITLE
Virtual Assistant

### DISCUSSION THREAD

**Discussion by DustinBrett** [DustinBrett/daedalOS#107] [Feature Requests]
I was thinking I could use https://www.smore.com/clippy-js for the UI and skies the limit for it's brain/voice/ears.

![image](https://user-images.githubusercontent.com/16656318/170892471-12b71a94-d5a2-4ae2-a5bf-61a9c341e439.png)

**Comment by tmunsch** [comment]
Make my dreams come true
